OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -522,15 +524,13 @@ static bool CanCreateGLWindow(GLProfile profile)
 			var window = SDL.SDL_CreateWindow("", 0, 0, 1, 1, flags);
 			if (window == IntPtr.Zero || !string.IsNullOrEmpty(SDL.SDL_GetError()))
 			{
-				SDL.SDL_ClearError();
```

Comment:
I'm almost certain that these were intentional, to stop error cases from leaking between profiles.